orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/ChromosomalRearrangementsDeterminer.java

```diff
@@ -0,0 +1,93 @@
+package com.hartwig.hmftools.orange.algo.purple;
+
+import java.util.List;
+
+import com.hartwig.hmftools.common.purple.PurpleCopyNumber;
+import com.hartwig.hmftools.common.genome.refgenome.RefGenomeCoordinates;
+import com.hartwig.hmftools.datamodel.orange.OrangeRefGenomeVersion;
+
+import org.jetbrains.annotations.NotNull;
+
+final class ChromosomalRearrangementsDeterminer
+{
+    private final RefGenomeCoordinates refGenomeCoordinates;
+
+    public ChromosomalRearrangementsDeterminer(@NotNull final OrangeRefGenomeVersion refGenomeVersion)
+    {
+        refGenomeCoordinates = getRefGenomeCoordinates(refGenomeVersion);
+    }
+
+    public static final double LOWER_THRESHOLD_TRISOMY = 2.8;
+    public static final double UPPER_THRESHOLD_TRISOMY = 3.5;
+    public static final double THRESHOLD_DELETION = 0.2;
+    public static final double PCT_98 = 0.98;
+    public static final double PCT_90 = 0.90;
```

Comment:
@kduyvesteyn Yes I did consider/try, but it's difficult to come up with a correct descriptive name, since e.g. 0.98 is used by both trisomy and codeletion logic and both percentages are lower limits.

Reply:
I would use 2 different variables for this actually.

The fact that they are both 0.98 is a detail, they represent different things (and you could imagine you may want to tweak one without touching the other, that would currently not be possible.